ref.html

```diff
@@ -156,11 +156,6 @@
       switched companies during edition) or to clarify that an editor is no longer active even if her
       name is still attached to the document out of respect for her past contributions.
     </dd>
-    <dt>w3cid</dt>
```

Comment:
Whoops, I should have checked _why_ it got removed instead of just assuming it was a change to `src/` that just hadn't been built yet. Sorry!